### ISSUE TITLE
4. Focus-stop Back to Top Link (Homepage template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#117]
### Betrokken pagina's

[Homepagina template](https://rijkshuisstijl-community-templates-3ix5516vd-nl-design-system.vercel.app/developer-overheid/homepage)

### Beschrijf het probleem

Na het klikken op de "Terug naar boven" link, gaat de volgende focus-stop naar de Vercel-toolbar, en de focus daarna gaat naar de adresbalk van de browser. De focus na het klikken op de link zou naar de bovenkant van de webpagina moeten gaan.

### Relevante code

```html
`<a class="rhc-page-subfooter__backtotop" href="#">Terug naar boven <span class="utrecht-icon" role="presentation" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-arrow-narrow-up "><path d="M12 5l0 14"></path><path d="M16 9l-4 -4"></path><path d="M8 9l4 -4"></path></svg></span></a>`
```

### WCAG-richtlijn

WCAG 2.4.3 Focus Order Level A

### Voeg een afbeelding toe (indien relevant)

![A tab stop widget shows that after clicking the Terug naar boven link, the focus does not go to the top of the page](https://github.com/user-attachments/assets/fd7f19c4-81e2-4638-9318-7899db08acce)

### To-Do's

ℹ Consistency: op iedere template pagina zowel een "Skip to Content" als "Back to Top" link
- belangrijk om deze op iedere pagina hetzelfde te tonen en laten functioneren (heb deze nu links boven de footer geplaatst - onderzoeken (Help Wanted) of dit uiteindelijk de beste plek is, mensen verwachten 'm vaak rechts bijvoorbeeld. Geen WCAG fail zoals het nu is - zo lang contrast goed is, focus + duidelijk wat het voor link is (SR leest voor: link, terug naar boven)

- [x] Focus na klik op Back to Top Link op de Skip Link plaatsen (`<body id="top"> `+ `href="#top"`)

- [x] Back to Top Link is (zichtbaar) focusbaar
(er zit wel verschil in focus ring op verschillende elementen op de pagina)
